src/fromager/wheels.py

```diff
@@ -84,6 +84,14 @@ def build_wheel(
     # TODO: refactor?
     # Build Rust without network access
     extra_environ["CARGO_NET_OFFLINE"] = "true"
+    # configure max jobs settings. should cover most of the cases, if not then the user can use ctx.jobs in their plugin
+    if ctx.jobs:
```

Comment:
Do we need a check for `> 0` or do we assume if they give a negative number something they're building knows what to do with it?

Reply:
Idea: check for `ctx.jobs is not None` and then map `ctx.jobs == 0` to `os.cpu_count()`.